test/net.test.js

```diff
@@ -12,39 +12,37 @@ describe("CALCULATE NET SALARY:", function () {
       test('Gross is equal 12.290503m', () => {
 
 
-        let netSalary = net(12290503, 0);
-        let roundedSalary = Math.round(netSalary);
+        let netSalary = net(12_290_503, 0, 1, new Date("2022-04-01"));
```

Comment:
Better to use `const`
`let`, `const` and `var` are three way to declare Js variable.
Nowadays, we avoid using var due to having issue.
`let` is used for changeable value, `const` is opposite. 
Check the rest as well